packages/crypto/src/networks/devnet/milestones.json

```diff
@@ -46,5 +46,9 @@
     {
         "height": 950000,
         "ignoreInvalidSecondSignatureField": false
+    },
+    {
+        "height": 1750000,
+        "vendorField255": true
```

Comment:
I would instead introduce a `vendorFieldLength` constant that defaults to `64/65` so we can increase the length at any time with a check for a single constant.